docker/importer/volume/surveillance-mediator/endpoint-resource-resolver-dhis.json

```
@@ -0,0 +1,100 @@
+{
+  "name": "Covid19 DHIS2 Orchestrator",
+  "description": "Endpoint for mapping the Covid19 case report into DHIS format and then forwarding it to DHIS2",
+  "endpoint": {
+    "pattern": "/resource-resolver-dhis"
+  },
+  "transformation": {
+    "input": "JSON",
+    "output": "JSON"
+  },
+  "inputValidation": {
+    "type": "object",
+    "properties": {
+      "requestBody": {
+        "type": "object"
+      }
+    },
+    "required": ["requestBody"]
+  },
+  "inputMapping": {
+    "requestBody.identifier": "teiIdentifier",
```

@bradsawadye 
Nesting
```suggestion
    "requestBody.data.identifier": "teiIdentifier",
```

---

addressed